kube-apiserver/kube-apiserver-deployment.yaml

```diff
@@ -12,54 +14,69 @@ spec:
       labels:
         app: kube-apiserver
     spec:
+      affinity:
+        podAntiAffinity:
+          requiredDuringSchedulingIgnoredDuringExecution:
+            - labelSelector:
+                matchExpressions:
+                  - key: app
+                    operator: In
+                    values: ["kube-apiserver"]
+              topologyKey: "kubernetes.io/hostname"
+            - labelSelector:
+                matchExpressions:
+                  - key: app
+                    operator: In
+                    values: ["kube-apiserver"]
+              topologyKey: "failure-domain.beta.kubernetes.io/zone"
       automountServiceAccountToken: false
       containers:
-      - name: kube-apiserver
```

Comment:
seems like the indentation was preturbed here. can you maintain the existing indentation so review is more straight forward?